packages/celotool/src/cmds/deploy/destroy/blockscout.ts

```diff
@@ -13,11 +13,9 @@ export const builder = {}
 export const handler = async (argv: DestroyArgv) => {
   await switchToClusterFromEnv()
 
-  const instanceName = `${argv.celoEnv}${fetchEnvOrFallback('BLOCKSCOUT_DB_SUFFIX', '')}`
-  const helmReleaseName = `${argv.celoEnv}-blockscout${fetchEnvOrFallback(
-    'BLOCKSCOUT_DB_SUFFIX',
-    ''
-  )}`
+  const dbSuffix = fetchEnvOrFallback('BLOCKSCOUT_DB_SUFFIX', '')
```

Comment:
Could you add `BLOCKSCOUT_DB_SUFFIX` to the `envVar` enum in `src/lib/env-utils.ts` and use that instead elsewhere as well?